README.md

```diff
@@ -67,6 +68,14 @@ func main() {
   	// Enable BLE interface.
 	must("enable BLE stack", adapter.Enable())
 
+	ctx, cancel := context.WithCancel(context.Background())
+	adapter.SetConnectHandler(func(device bluetooth.Device, connected bool){
+		if connected {
+			println("device connected:", device.Address.String(), device.RSSI, device.LocalName())
```

Comment:
```suggestion
			println("device connected:", device.Address.String())
```

Neither `device.RSSI` nor `device.LocalName()` exist, which causes an error on my machine:

```
examples/advertisement/main.go:18:65: device.RSSI undefined (type bluetooth.Device has no field or method RSSI)
examples/advertisement/main.go:18:78: device.LocalName undefined (type bluetooth.Device has no field or method LocalName)
```

Reply:
After I changed the line of code to the above, it did work for me.

```shell
$ go run ./examples/advertisement/
advertising...
device connected: 6A:20:3A:EC:0A:B6
```